udata_front/theme/gouvfr/datagouv-components/src/components/ReuseCard/ReuseCard.vue

```diff
@@ -0,0 +1,100 @@
+                </AppLink>
+                <OrganizationNameWithCertificate v-else :organization="reuse.organization" />
+              </span>
+            </template>
+            <span class="dash-after-sm" v-else>
+              {{ ownerName }}
+            </span>
+            {{ t('Published {date}', {date: formatRelativeIfRecentDate(reuse.created_at)}) }}
+          </p>
+          <div class="fr-grid-row fr-grid-row--middle fr-mt-1v fr-mr-1v">
+            <span class="dash-after-sm">
+              {{ reuseType }}
+            </span>
+            <p class="fr-text--sm fr-my-0">
+              <span class="fr-icon-star-line fr-icon--sm fr-px-1v" aria-hidden="true"></span>{{ reuse.metrics.followers }}
+            </p>
+            <p class="fr-text--sm fr-my-0">
+              <span class="fr-icon-eye-line fr-icon--sm fr-px-1v" aria-hidden="true"></span>{{ reuse.metrics.views }}
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="fr-card__header">
+      <div class="absolute top-0 fr-mt-2w fr-ml-2w">
+        <p class="fr-badge fr-badge--sm fr-badge--mention-grey text-grey-380" v-if="reuse.private">
+          <span class="fr-icon-lock-line fr-icon--sm">
+            {{ t('Private') }}
+          </span>
+        </p>
+      </div>
+      <Placeholder
+        class="fr-responsive-img ratio-sm"
+        type="reuse"
+        :src="reuse.image"
+        :size="80"
+      />
+    </div>
+  </article>
+</template>
+
+<script lang="ts">
+export type ReuseProps = {
+  reuse: Reuse,
+
+  /**
+   * The reuseUrl is a route location object to allow Vue Router to navigate to the details of a reuse.
+   * It is used as a separate prop to allow other sites using the package to define their own reuse pages.
+   */
+  reuseUrl: RouteLocationRaw,
+
+  /**
+   * The organizationUrl is an optional route location object to allow Vue Router to navigate to the details of the organization linked to tha reuse.
+   * It is used as a separate prop to allow other sites using the package to define their own organization pages.
+   */
+  organizationUrl?: RouteLocationRaw
+};
+</script>
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n';
+import Placeholder from '../utils/Placeholder.vue';
+import { OrganizationNameWithCertificate } from "../Organization";
+import { truncate } from "../../helpers";
+import type { Reuse } from '../../types/reuses';
+import { formatRelativeIfRecentDate } from '../../helpers';
+import { useOwnerName } from '../../composables';
+import useReuseType from '../../composables/useReuseType';
+import AppLink from "../AppLink/AppLink.vue";
+import { RouteLocationRaw } from 'vue-router';
+
+const props = defineProps<ReuseProps>();
+
+const { t } = useI18n();
+
+const ownerName = useOwnerName(props.reuse);
+const { label: reuseType } = useReuseType(() => props.reuse.type);
+</script>
+
+<style>
+.ratio-sm {
+  aspect-ratio: 1.4;
```

Comment:
Is there a reason to add this aspect-ratio ? I don't think we have an exact aspect-ratio on our different uses of reuse images